gen/aircraft.py

```diff
@@ -1357,11 +1377,27 @@ def configure_awacs(
     ) -> None:
         group.task = AWACS.name
         self._setup_group(group, AWACS, package, flight, dynamic_runways)
+
+        freq = self.radio_registry.alloc_uhf()
+
         self.configure_behavior(
             group,
             react_on_threat=OptReactOnThreat.Values.EvadeFire,
             roe=OptROE.Values.WeaponHold,
             restrict_jettison=True,
+            do_aewc=True,
+            freq=freq
+        )
+
+        arrival = package.time_over_target + timedelta(hours=4.5)
```

Comment:
Should come from the `AwacsFlightPlan`. Add `start_time` and `end_time` properties to that, and here do:

```
if not isinstance(flight.flight_plan, AwacsFlightPlan):
    logging.error(f"Cannot configure AEW&C tasks for {flight} because it does not have an AEW&C flight plan.")
    return

self.air_support.awacs.append(AwacsInfo(
    ...
    start_time=flight.flight_plan.start_time,
    end_time=flight.flight_plan.end_time
))
```

That way if we decide that the start/end times of the task should be tied to the orbit waypoint it's easy to tweak.